src/Tokenizer/Token.php

```diff
@@ -67,6 +67,17 @@ public function __construct($token)
         }
     }
 
+    public function __debugInfo()
+    {
+        return array(
+            "\0+\0content" => $this->content,
```

Comment:
what php are you using ?
https://3v4l.org/nvjvI

here, the output is totally different, and between `\0` is the name of the class which declared a property